programs/marginfi/src/instructions/marginfi_group/liquid_insurance_fund/create_fund.rs

```diff
@@ -0,0 +1,203 @@
+use crate::{
+    constants::{
+        INSURANCE_VAULT_AUTHORITY_SEED, INSURANCE_VAULT_SEED, LIQUID_INSURANCE_MINT_AUTHORITY_SEED,
+        LIQUID_INSURANCE_MINT_METADATA_SEED, LIQUID_INSURANCE_MINT_SEED,
+    },
+    events::{LiquidInsuranceFundEventHeader, MarginfiCreateNewLiquidInsuranceFundEvent},
+    state::{liquid_insurance_fund::LiquidInsuranceFund, marginfi_group::Bank},
+    MarginfiGroup, MarginfiResult,
+};
+use anchor_lang::prelude::*;
+use anchor_spl::token::{Mint, Token, TokenAccount};
+
+use mpl_token_metadata::state::DataV2;
+
+use anchor_spl::metadata::{create_metadata_accounts_v3, CreateMetadataAccountsV3, Metadata};
+
+#[derive(Accounts)]
+#[instruction(
+    params: InitMintParams,
+    min_withdraw_period: u64,
+)]
+pub struct CreateNewLiquidInsuranceFund<'info> {
+    pub marginfi_group: AccountLoader<'info, MarginfiGroup>,
+
+    #[account(
+        mut,
+        address = marginfi_group.load()?.admin,
+    )]
+    pub marginfi_authority: Signer<'info>,
+
+    #[account(mut)]
+    pub signer: Signer<'info>,
+
+    #[account(
+        init,
+        space = 8 + std::mem::size_of::<LiquidInsuranceFund>(),
+        payer = signer,
+        seeds = [
+            marginfi_group.key().as_ref(),
```

Comment:
in case there are any other features down the line that use group & bank, I think we should switch this out for it's own `LIQUID_INSURANCE_FUND` seed. In other words, i'd follow a similar derivation path as all other accounts in this context